### ISSUE TITLE
maintained?

### ISSUE THREAD

**Issue by gfredericks** [gfredericks/clomp#1]
I'm in need of a stomp client and I noticed that:
1. there don't seem to be any besides this one
2. there's been no activity on this repo in over four years, from which I infer that nobody is using it

I did a quick test in my repl and it seems to work correctly, so I was planning on upgrading it for clojure 1.7 and creating a fixed release (rather than a snapshot). I was going to do this by forking the repo and using a separate group name for releases, but I was wondering if I should try to push changes back here as pull requests. I.e., are you interested in maintaining this library, or should I plan on my fork being permanent?

**Comment by ninjudd** [comment]
I don’t have time to maintain anything that I’m not using directly these days, so a permanent fork sounds good to me. I’m also happy to transfer the repo to you if you’d prefer that.

**Comment by gfredericks** [comment]
repo transfer would move it to `/gfredericks/clomp`, correct? is there any tangible difference (from just using a forked repo) besides some subtle github things about forks being linked to parents or something?

**Comment by ninjudd** [comment]
That's right. The main advantage of a repo transfer is that it makes it clear which repo is maintained. Can do that with the README though too.

**Comment by gfredericks** [comment]
And then `/ninjudd/clomp` will just redirect won't it? I guess that sounds better. Thanks!

**Comment by gfredericks** [comment]
baaaaaah. github sent me an email about the transfer a week ago, which I finally got around to looking at, and noticed that it said it expires after a day :(.

If you do it again, I'll process it right away this time. Sorry.

**Comment by ninjudd** [comment]
No worries. Resent.
